src/SSHLibrary/javaclient.py

```diff
@@ -46,7 +46,8 @@ def _login(self, username, password, look_for_keys='ignored'):
         if not self.client.authenticateWithPassword(username, password):
             raise SSHClientException
 
-    def _login_with_public_key(self, username, key_file, password):
+    def _login_with_public_key(self, username, key_file, password,
+                               allow_agent='ignored', look_for_keys='ignored'):
```

Comment:
I wonder should we raise an exception if these arguments are used on Jython instead of just ignoring them? Should then do the same with `_login` as well. I'm fine with just ignoring and if we were to change it, at least with `_login` that shouldn't be part of this PR.

Reply:
Added an exception after all. Even though it is mentioned in documentation, I think it's better to have this case handled in code also. As for the `_login` method, I think we should also change it to throw the exception in separate PR, before the release maybe?

Reply:
Sounds good to me.